io.elementary.Sdk.json.in

```diff
@@ -59,8 +58,43 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "https://github.com/elementary/stylesheet.git",
-                    "tag": "5.4.2"
+                    "url": "https://github.com/elementary/stylesheet.git"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "sassc",
+                    "cleanup": [ "*" ],
+                    "sources": [
+                        {
+                            "type": "archive",
```

Comment:
one question, is there a reason to not use git here?

Reply:
Not really, mostly cargo culted the module. Will look at using git.

Reply:
Should be good now.